src/alpaca_eval/models_configs/openbuddy-falcon-40b-v9/configs.yaml

```diff
@@ -0,0 +1,13 @@
+openbuddy-falcon-40b-v9:
+  prompt_template: "openbuddy-falcon-40b-v9/prompt.txt"
+  fn_completions: "huggingface_local_completions"
+  completions_kwargs:
+    model_name: "./openbuddy-falcon-40b-v9-bf16" # local path
+    model_kwargs:
+      torch_dtype: 'bfloat16'
+    max_new_tokens: 2048
+    temperature: 0.7
+    top_p: 1.0
+    do_sample: True
+  pretty_name: "OpenBuddy-Falcon-40b-v9"
+  link: "https://huggingface.co/OpenBuddy/openbuddy-falcon-40b-v9-bf16"

```

Comment:
This link is 404. If it's not publicly available, let's remove this link for now. 
Or you may want to link people to the form they can fill out to download the model.

Reply:
Thanks for pointing out!

I have changed the link to our Early Access organization page, and there's a guide for how to apply for the model.